scripts/build_docker_image.sh

```diff
@@ -70,9 +66,6 @@ else
 fi
 
 VM_ID=${VM_ID:-"${DEFAULT_VM_ID}"}
-if [[ "${VM_ID}" != "${DEFAULT_VM_ID}" ]]; then
-  DOCKERHUB_TAG="${VM_ID}-${DOCKERHUB_TAG}"
```

Comment:
not sure if this was needed at all.

Reply:
It looks like the vm id can be set by a dispatch event, maybe we should keep this?